assets/scss/base/_buttons.scss

```diff
@@ -32,6 +33,24 @@ button, .button {
   @include core-19();
 }
 
+.button--blue {
+  @extend .button--get-started;
+  @include button($govuk-blue-colour);
+  @include bold-27();
+  width:100%;
```

Comment:
But maybe a `.button--full-width` should be created to handle the positional and geometric styles?